pkg/common/health/logger.go

```diff
@@ -0,0 +1,33 @@
+package health
+
+import (
+	"github.com/InVisionApp/go-health"
+	"github.com/prometheus/common/log"
+	"github.com/sirupsen/logrus"
+)
+
+// statusListener logs
+type statusListener struct {
+	log logrus.FieldLogger
+}
+
+// Assert statusListener implements IStatusListener
+var _ health.IStatusListener = &statusListener{}
+
+// HealthCheckFailed is triggered when a health check fails the first time
+func (sl *statusListener) HealthCheckFailed(entry *health.State) {
+	log.With("check", entry.Name).
+		With("details", entry.Details).
+		With("error", entry.Err).
+		Warn("Health check failed")
+}
+
+// HealthCheckRecovered is triggered when a health check recovers
+func (sl *statusListener) HealthCheckRecovered(entry *health.State, recordedFailures int64, failureDurationSeconds float64) {
+	log.With("check", entry.Name).
```

Comment:
sl.log...

Reply:
Fixed these two.  This was also dragging in an extra dep on promethus/common/log, intead of using logrus directly.